src/spawn-win.c

```diff
@@ -1,10 +1,75 @@
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <windows.h> 
+#include <strsafe.h>
+
 #include "include/mini/common.h"
 
+static char executable_path[BUFSIZE];
+
+int popen_complete(const char* cmd)
+{
```

Comment:
Style, please put `{` on the same line as `popen_complete`.

Let's make `popen_complete` static.